analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -146,45 +146,45 @@ class Gradle : PackageManager() {
             // Only look for a package when there was no error resolving the dependency.
             packages.getOrPut("${dependency.groupId}:${dependency.artifactId}:${dependency.version}") {
                 if (dependency.pomFile.isNotBlank()) {
-                    val projectBuildingResult = maven.buildMavenProject(File(dependency.pomFile))
-                    projectBuildingResult.project.let {
-                        return@getOrPut Package(
+                    with(maven.buildMavenProject(File(dependency.pomFile))) {
```

Comment:
Isn't a "project ->" missing at the end of the line? I just wonder where the name "project" comes from below.

Reply:
Ok, I found out myself.